tests/data/nyaml2nxdl/Ref_NXellipsometry-docCheck.nxdl.xml

```diff
@@ -66,7 +66,7 @@
          - https://www.angstromadvanced.com/resource.asp
          - https://pypolar.readthedocs.io/en/latest/
     </doc>
-    <group type="NXentry">
+    <group type="NXentry" optional="false">
```

Comment:
should not the optionality be skipped in this nxdl.xml file?

Reply:
Removed!